deploy/crds/infra.watch_v1alpha1_servicetelemetry_cr.yaml

```diff
@@ -8,4 +8,6 @@ spec:
   highAvailabilityEnabled: false
   storageEphemeralEnabled: false
   graphingEnabled: false
+  snmpTrapsEnabled: false
+  snmpTrapsTarget: 192.168.24.254
```

Comment:
This should show up in the 1.1.0 examples if you generated the CSV :)

Reply:
done